### DISCUSSION TITLE
Support `singleAttributePerLine` similar to how prettier does

### DISCUSSION THREAD

**Discussion by nahoc** [biomejs/biome#698] [Ideas]
Love biome, the only thing preventing us to fully adopt it at my job is the missing `singleAttributePerLine` feature from prettier, example:

```tsx
<Stack
  direction="row"
  alignItems="center"
  justifyContent="space-between"
  gap={2}
  sx={{ mb: 2 }}
>
```
vs
```tsx
<Stack direction='row' alignItems='center' justifyContent='space-between' gap={2} sx={{ mb: 2 }}>
```

I'm aware of the philosophy behind the biome formatter that aims to keep minimal features, but ideally it would have full parity with existing prettier feature. Thank you for this amazing tool!

**Comment by tylerlaprade** [comment]
Same here, this is the only thing blocking my adoption. This format is so much more readable than having all attributes on one line.

**Reply by octoshikari** [reply]
+1. I'm waiting for this rule for full migration into the biome.

**Comment by ematipico** [comment]
Closing. If someone wants to implement the feature, here's the issue: https://github.com/biomejs/biome/issues/1706

**Comment by rujorgensen** [comment]
Would this work on function arguments in js/ts too 🙂? Whenever I hear mention about this option, I only see examples in html.
